core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -246,6 +247,15 @@ object ConfigCommand extends Config {
 
   private[admin] def parseConfigsToBeAdded(opts: ConfigCommandOptions): Properties = {
     val props = new Properties
+    if (opts.options.has(opts.addConfigFile)) {
+      val file = opts.options.valueOf(opts.addConfigFile)
```

Comment:
What happens if the provided file does not exist? It would be great if we could handle this gracefully and error our to the user.

Reply:
It's currently a FNFException, which I agree is not the best user experience. I'll wrap this in an IllegalArgumentException so that it doesn't print the stack trace.

```Error while executing config command with args '--bootstrap-server localhost:9092 --entity-type brokers --entity-default --alter --add-config-file new.properties'
java.io.FileNotFoundException: new.properties (No such file or directory)
        at java.io.FileInputStream.open0(Native Method)
        at java.io.FileInputStream.open(FileInputStream.java:195)
        at java.io.FileInputStream.<init>(FileInputStream.java:138)
        at kafka.admin.ConfigCommand$.parseConfigsToBeAdded(ConfigCommand.scala:252)
        at kafka.admin.ConfigCommand$.alterConfig(ConfigCommand.scala:315)
        at kafka.admin.ConfigCommand$.processCommand(ConfigCommand.scala:302)
        at kafka.admin.ConfigCommand$.main(ConfigCommand.scala:92)
        at kafka.admin.ConfigCommand.main(ConfigCommand.scala)
```